src/content/docs/concepts/layout.md

```diff
@@ -6,7 +6,7 @@ The coordinate system in Ratatui runs left to right, top to bottom, with the ori
 top left corner of the terminal. The x and y coordinates are represented by u16 values and are
 generally listed in that order in most places.
 
-```kroki type=svgbob
+```svgbob
```

Comment:
This is just rendering as ASCII art, not SVG right?

Reply:
Yes, but will add a svgbob plugin in a separate PR.